src/main/java/edu/wright/airviewer2/AIRViewerController.java

```diff
@@ -278,6 +297,16 @@ private AIRViewerModel reinitializeWithModel(AIRViewerModel aModel) {
                 refreshUserInterface();
                 return pageImageGroup;
             });
+
+            saveMenuItem.setOnAction((ActionEvent event) -> {
+                try {
+                    model.save(new File(path));
+                    MessageBox.show("Saved pdf at " + path, "Saved!");
+                } catch (Exception e) {
+                    System.out.println(e);
+                    MessageBox.show(e.toString(), "IO Exception");
+                }
```

Comment:
incase of exception instead of showing the raw exception can we display some user-friendly message on the UI to the user.

Reply:
Pushed the changes. Please review it.